utmap-server/README.html

```diff
@@ -302,10 +294,15 @@ <h3 class="field-label">Example data</h3>
   "name" : "Game Night",
   "description" : "Come join us for Cards Against Humanity and Super Smash Bros.",
   "startTime" : "2021-09-20T17:00:00Z",
+  "location" : {
+    "code" : "DH",
+    "name" : "Deerfield Hall",
+    "_id" : "d290f1ee-6c54-4b01-90e6-d701748f0851",
+    "coords" : "200, 400"
```

Comment:
Might be better not to have coords, because if we use a map API we don’t know what kind of data we will be passing yet. I think it’s better not to implement stuff for the map, and to add this as a modification later

Reply:
Alright, I can remove it